main.js

```diff
@@ -0,0 +1,66 @@
+var players = [];
```

Comment:
var is a hold over from the olden days of JavaScript, and can have some weird hosting behaviors.  It doesn't really make a difference here since these are all in the global scope, but you should use either const or let to declare your variables to get in the habit of using them/